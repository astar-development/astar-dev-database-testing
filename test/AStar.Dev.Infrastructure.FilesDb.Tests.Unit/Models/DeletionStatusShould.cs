using AStar.Dev.Utilities;
using Shouldly;

namespace AStar.Dev.Infrastructure.FilesDb.Models;

public class DeletionStatusShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new DeletionStatus
           {
               SoftDeleted       = new DateTimeOffset(2025, 7, 28, 1, 2, 3, TimeSpan.Zero),
               HardDeletePending = new DateTimeOffset(2025, 7, 28, 2, 3, 4, TimeSpan.Zero),
               SoftDeletePending = new DateTimeOffset(2025, 7, 28, 3, 4, 5, TimeSpan.Zero)
           }
           .ToJson()
           .ShouldMatchApproved();
}
