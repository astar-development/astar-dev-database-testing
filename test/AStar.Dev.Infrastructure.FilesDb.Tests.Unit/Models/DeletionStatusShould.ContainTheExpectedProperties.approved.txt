{
  "softDeleted": "2025-07-28T01:02:03+00:00",
  "softDeletePending": "2025-07-28T03:04:05+00:00",
  "hardDeletePending": "2025-07-28T02:03:04+00:00"
}